vNext/extensions/applicationinsights-react-js/README.md

```diff
@@ -3,20 +3,86 @@
 [![Build Status](https://travis-ci.org/Microsoft/ApplicationInsights-JS.svg?branch=master)](https://travis-ci.org/Microsoft/ApplicationInsights-JS)
 [![npm version](https://badge.fury.io/js/%40microsoft%2Fapplicationinsights-react-js.svg)](https://badge.fury.io/js/%40microsoft%2Fapplicationinsights-react-js)
 
+React Plugin for the Application Insights Javascript SDK, enables the following:
+
+- Tracking of router changes
+- React components usage statistics
+
 React Plugin for the Application Insights Javascript SDK
 
-## Build:
-```
-npm install -g grunt-cli
-npm install
-npm run build --silent
-```  
+## Getting Started
+
+Install npm package:
 
-## Run unit tests:
+```bash
+npm install --save @microsoft/applicationinsights-react-js
```

Comment:
the save flag is no longer necessary since npm 5, which was released 2 years ago, so I think it can be dropped https://blog.npmjs.org/post/161081169345/v500